.travis.yml

```diff
@@ -14,8 +14,7 @@ env:
   global:
     - DOTNET_SKIP_FIRST_TIME_EXPERIENCE: true
     - DOTNET_CLI_TELEMETRY_OPTOUT: 1
-mono:
-  - 4.0.5
+mono: none
```

Comment:
@Eilon was mentioning earlier today that repos like EF, Configuration, Options, (Logging?) etc. can be used in workloads other than ASP.NET and still need to tested on Mono.